src/Microsoft.Azure.SignalR.Common/ServiceMessages/RuntimeServicePingMessages.cs

```diff
@@ -97,6 +99,28 @@ public static bool TryGetClientCount(this ServicePingMessage ping, out int clien
             }
             clientCount = count;
             return true;
+        }   
+        
+        public static bool TryGetServerCount(this ServicePingMessage ping, out int serverCount)
```

Comment:
Naming to `TryGetServerConnectionCount()` would be more accurate, since the another `servers` ping is trying to get server count...